specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1930,6 +2076,100 @@
         }
       },
       "description": "The backup key result, containing the backup blob."
+    },
+    "KeyPolicy": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The key policy id.",
+          "readOnly": true
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeActions"
+          },
+          "description": "Actions that will be performed by Key Vault over the lifetime of a key."
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyPolicyAttributes",
+          "description": "The key policy attributes."
+        }
+      },
+      "description": "Management policy for a key."
+    },
+    "LifetimeActions": {
```

Comment:
Are lifetime actions the same as for certificates? If so and by design that they always will be, perhaps move the definitions to common.json and ref them both in certificates.json and here in keys.json to avoid differences. This should help avoid them getting out of sync, and may improve supportability in some SDKs.